.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -2,6 +2,6 @@
 blank_issues_enabled: false
 contact_links:
 #...or links in general. Makes sure people stay to the topic on FreeFlarum.
-  - name: Flarum Discuss
+  - name: Have questions about Flarum in general?
```

Comment:
@CWkevo We at the  Pins team mostly ask questions to the users as issue template titles, like `Found a security vulnerability?` or `Have any feedback about our docs?`.

Reply:
That's okay, but someone might be confused why it redirects them somewhere else. I'd name it "Flarum Discuss" instead, and also we've got a notice about where to ask Flarum questions in our extension template already.

Reply:
@CWKevo Thanks for the feedback! I'll revert it to `Flarum Discuss` and we'll use your feedback to improve our GitHub issue templates.